charts/zqd/values.yaml

```diff
@@ -0,0 +1,87 @@
+# Default values for zqd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+datadir: "/spaces"
+
+configyaml: false
+
+replicaCount: 1
+
+image:
+  repository: "localhost:5000/"
+  # when not using docker registry:2 use repository: "docker.io/library/"
+  pullPolicy: Always
+  # have to use this pull policy if using a latest tag
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "zqd"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: 
+  # request a linkerd sidecar (works if Linkerd is installed)
+  linkerd.io/inject: enabled
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
```

Comment:
Do we want the commented out values in this file?

Reply:
I went ahead and kept a lot of the boilerplate template that is generated by "helm create" -- which includes these commented out lines. Some of it, like the HPA support, we will need for later, so I will leave the values.yaml in its mostly vanilla state for now.